domain/secretbackend/service/package_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	gc "gopkg.in/check.v1"
 )
 
+//go:generate go run go.uber.org/mock/mockgen -typed -package service -destination atomic_mock_test.go github.com/juju/juju/domain AtomicContext
```

Comment:
Delete me and use the new test helper

Reply:
Won't be needed anyway since we don't need run atomic now due to the refactoring.